nuxeo-services/nuxeo-platform-filemanager-api/src/main/java/org/nuxeo/ecm/platform/filemanager/api/FileManager.java

```diff
@@ -83,10 +83,24 @@ DocumentModel createDocumentFromBlob(CoreSession documentManager, Blob input, St
      * @param fullname the full name of the folder
      * @param path the path were to create the folder
      * @return the Folder Created
+     * @deprecated since 9.1, use {@link #createFolder(CoreSession, String, String, boolean)} instead
      */
+    @Deprecated
     DocumentModel createFolder(CoreSession documentManager, String fullname, String path) throws IOException;
 
     /**
+     * Creates a Folder.
+     *
+     * @param fullname the full name of the folder
+     * @param path the path were to create the folder
+     * @param overwrite boolean how decide to overwrite or not
```

Comment:
`@param overwrite whether to overwrite an existing folder with the same title or not`